crates/autopilot/src/domain/settlement/trade/mod.rs

```diff
@@ -154,17 +148,19 @@ pub struct Jit {
 /// Fee per trade in a solution. These fees are taken for the execution of the
 /// trade.
 #[derive(Debug, Clone)]
-pub struct ExecutedFee {
-    /// Gas fee spent to bring the order onchain
-    pub network: eth::SellTokenAmount,
+pub struct FeeBreakdown {
+    /// Total fee spent to bring the order onchain (network fee + protocol fee)
```

Comment:
nit: not sure the protocol fee is spent to bring the order onchain 

```suggestion
    /// Total fee the trade was charged (network fee + protocol fee)
```